### PR TITLE
Bugfix/25 building via mvn clean install requires config server to run

### PR DESCRIPTION
**Pull request by BrunoSilvaATB**
Now CoatRack doesn't need a config server to run
closes coatrack#25

### CODE REVIEW

**Comment by BrunoSilvaATB** [comment]
Now both application-test.yml and the bootstrap.properties are cleaned up

File: spring-boot/admin/src/test/resources/bootstrap.properties [hunk 1]
```diff
@@ -0,0 +1,8 @@
+spring.application.name=ygg-admin
+spring.cloud.config.uri=https://config.coatrack.eu
```
Comment (gr-hovest-atb) [inline spring-boot/admin/src/test/resources/bootstrap.properties:2]: To make it easier understandable for the reader, maybe we should make clear that these are dummy values for the test. E.g.

```suggestion
spring.cloud.config.uri=https://test-dummy-config-url.coatrack.eu
```

File: spring-boot/admin/src/test/resources/bootstrap.properties [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+spring.cloud.config.username= admin
```
Comment (gr-hovest-atb) [inline spring-boot/admin/src/test/resources/bootstrap.properties:3]:
```suggestion
spring.cloud.config.username=dummyTestUsername
```

File: spring-boot/admin/src/test/resources/application-test.yml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    call-request-fee: 0
+    database:
+      insertSampleDataOnStartup: true
+    gettingStarted:
+      consumer:
+        testService:
+          provider.username: BrunoSilvaATB
+          uriIdentifier: getting-started-service-consumer
+  mail:
+    verification.server.url: http://localhost:8080
```
Comment (gr-hovest-atb) [inline spring-boot/admin/src/test/resources/application-test.yml:33]: To allow the build to run without specific local mvn settings, we should also add the following here:

` sender.user: example-mail@example.com`
` sender.password: examplePassword`

File: spring-boot/admin/src/test/resources/application-test.yml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+      fix: 1
+      bookKeeping.contact: email@example.com
+    api-base-url-for-gateway: https://coatrack.eu/api/
+    call-request-fee: 0
+    database:
+      insertSampleDataOnStartup: true
+    gettingStarted:
+      consumer:
+        testService:
+          provider.username: BrunoSilvaATB
```
Comment (gr-hovest-atb) [inline spring-boot/admin/src/test/resources/application-test.yml:30]: Maybe put a dummy user name?

File: spring-boot/admin/src/test/resources/application-test.yml [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+    nl: DEBUG
+    org: INFO
+    com: DEBUG
+ygg:
+  admin:
+    server:
+      url: http://localhost:8080
+    servicecovers:
+      path: /tmp
+      url: http://localhost/coatrack-servicecovers/
```
Comment (gr-hovest-atb) [inline spring-boot/admin/src/test/resources/application-test.yml:17]: General comment: there are several occurrences of `localhost` in this file, maybe replace them by something like `example-url.coatrack.eu`

File: spring-boot/admin/src/test/resources/bootstrap.properties (hunk 1 above)
Comment (BrunoSilvaATB) [reply spring-boot/admin/src/test/resources/bootstrap.properties:2]: Done, and it worked well

File: spring-boot/admin/src/test/resources/bootstrap.properties (hunk 2 above)
Comment (BrunoSilvaATB) [reply spring-boot/admin/src/test/resources/bootstrap.properties:3]: Done and working well

File: spring-boot/admin/src/test/resources/application-test.yml (hunk 4 above)
Comment (BrunoSilvaATB) [reply spring-boot/admin/src/test/resources/application-test.yml:30]: Done and working well

File: spring-boot/admin/src/test/resources/application-test.yml (hunk 5 above)
Comment (BrunoSilvaATB) [reply spring-boot/admin/src/test/resources/application-test.yml:17]: Done and Working well

File: spring-boot/admin/src/test/resources/application-test.yml (hunk 3 above)
Comment (BrunoSilvaATB) [reply spring-boot/admin/src/test/resources/application-test.yml]: Done and working well.

**Comment by BrunoSilvaATB** [comment]
Update master, and test this bugfix with the script

**Comment by BrunoSilvaATB** [comment]
The 2 actions taken here were:
- Suggestions Applied (created a build script for windows systems)
- The branch was updated with the current master

 The build tested and working:
image

**Comment by BrunoSilvaATB** [comment]
Branch reverted (this branch was successfully reverted to the state before merging with master)

**Comment by BrunoSilvaATB** [comment]
This branch is updated with the latest master version.
This branch still needs to be tested after the bugfix for the ResourceServerProperties is merged

**Comment by BrunoSilvaATB** [comment]
@gr-hovest-atb as now we are removing the config server, is this PR still important?
As the objective was to be able to run CoatRack without config server

**Comment by BrunoSilvaATB** [comment]
reminder: Test running master branch of CoatRack without internet connection/ spring boot connection to master, and if this works without config server then close this PR

**Comment by gr-hovest-atb** [comment]
I did a test of the current master and it seems like this issue was implicitly solved by coatrack#158, which removes the "runtime config server dependency" from admin

**Comment by BrunoSilvaATB** [comment]
Based on the testing from @gr-hovest-atb (check comment above), this issue is fixed

**Pull request closed without merging**